File: dotnet/src/Wata.Sdk/Errors/WataWebhookException.cs
namespace Wata.Sdk.Errors;

/// <summary>Подпись вебхука не сошлась, ключ не удалось получить или разобрать.</summary>
public sealed class WataWebhookException : WataException
{
    public WataWebhookException(string message, Exception? innerException = null)
        : base(message, innerException)
    {
    }
}

File: java/src/main/java/pro/wata/sdk/errors/WataWebhookException.java
package pro.wata.sdk.errors;

/** Подпись вебхука не сошлась, либо не удалось получить публичный ключ. */
public class WataWebhookException extends WataException {

    public WataWebhookException(String message) {
        super(message);
    }

    public WataWebhookException(String message, Throwable cause) {
        super(message, cause);
    }
}
